locale/de/LC_MESSAGES/meerk40t.po

```diff
@@ -2374,68 +2394,68 @@ msgstr "Öffnen"
 
 #: meerk40t/gui/panes/toolbarproject.py:60 meerk40t/gui/wxmeerk40t.py:1193
 msgid "Simulate"
-msgstr ""
+msgstr "Simulieren"
 
 #: meerk40t/gui/panes/toolbarproject.py:104 meerk40t/gui/wxmeerk40t.py:1219
 #: meerk40t/gui/wxmeerk40t.py:2094
 msgid "Preparing simulation..."
-msgstr ""
+msgstr "Bereite Simulation vor..."
 
 #: meerk40t/gui/panes/toolbarcontrol.py:121 meerk40t/gui/wxmeerk40t.py:1238
 msgid "Control"
-msgstr ""
+msgstr "Steuerung"
 
 #: meerk40t/gui/panes/toolbarpreferences.py:97 meerk40t/gui/wxmeerk40t.py:1315
 msgid "Preferences"
 msgstr "Setup"
 
 #: meerk40t/gui/panes/toolbarpreferences.py:39 meerk40t/gui/wxmeerk40t.py:1333
 msgid "Config"
-msgstr ""
+msgstr "Konfiguration"
 
 #: meerk40t/gui/panes/toolbarpreferences.py:65 meerk40t/gui/wxmeerk40t.py:1352
 msgid "Keymap"
 msgstr "Tastenkürzel"
 
 #: meerk40t/gui/wxmeerk40t.py:1704
 msgid "&New\tCtrl-N"
-msgstr ""
+msgstr "&Neu\tStrg-N"
 
 #: meerk40t/gui/wxmeerk40t.py:1705
 msgid "&Open Project\tCtrl-O"
-msgstr ""
+msgstr "&Oeffne Pr&ojekt\tStrg-O"
 
 #: meerk40t/gui/wxmeerk40t.py:1707
 msgid "&Recent"
-msgstr ""
+msgstr "&Kürzlich"
 
 #: meerk40t/gui/wxmeerk40t.py:1708
 msgid "&Import File"
-msgstr ""
+msgstr "&Importiere Datei"
 
 #: meerk40t/gui/wxmeerk40t.py:1710
 msgid "&Save\tCtrl-S"
-msgstr ""
+msgstr "&Speichern\tStrg-S"
 
```

Comment:
The `tCtrl` bits might not translate since they are for setting up international hotkeys. And may actually need to say Ctrl to work.

Reply:
@jpirnay According to https://docs.wxpython.org/wx.MenuItem.html#wx.MenuItem.SetItemLabel @tatarize is correct to say that "Ctrl" should not be translated to "Strg".

This needs to be corrected before this can be merged. And this will give you time for the other few changes you want to make.